src/e2e-test/features/selectcolumnpanel.feature

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Integration_Tests
+Feature: SelectColumnPanel
+
+  @SelectColumnPanel
+  Scenario: Go through the SelectColumn Panel functionality
```

Comment:
Tests here should:
* Verify that a column for a single-column directive can be applied
* Verify that 2 columns for a 2-column directive can be applied
* Verify that multiple columns for a >2-column directive can be applied

Reply:
Each item should be its own scenario

Reply:
These scenarios will be covered in Keep, Copy and Delete column transformation feature (PR link: https://github.com/cdapio/cdap-ui/pull/811)